subtests/docker_cli/run_cgroups/run_cgroups.py

```diff
@@ -55,3 +56,42 @@ class memory_no_cgroup(memory_base):
 
 class memory_negative(memory_base):
     pass
+
+
+class cgroup_parent_invalid_name(cgroup_parent_base):
+    """
+    Invalid value for --cgroup-parent option
+    """
+
+    def initialize(self):
+        super(cgroup_parent_invalid_name, self).initialize()
+        self._setup("/foobar",
+                    "",
+                    "Error response from daemon: Cannot start container {cid}:"
+                    " [8] System error: Invalid slice name /foobar")
```

Comment:
Reading more, no I think what you have here with the call to _setup() is fine.  There are many other tests that rely on shared initialization functions and overriding ``initialize()``.  No prob. doing it this way.